DS implementations/STACKS/Stack_Implementation.cpp

```diff
@@ -0,0 +1,91 @@
+/* C++ program to implement basic stack operations */
+#include <bits/stdc++.h>
+
+using namespace std;
+
+#define MAX 1000
+
+class Stack
+{
+    int top;
+
+public:
+    int a[MAX]; // Maximum size of Stack
+
+    Stack()
+    {
+        top = -1;
+    }
+
+    bool push(int x);
+    int pop();
+    int peek();
+    bool isEmpty();
+};
+
+bool Stack::push(int x)
```

Comment:
Return a default value, will throw compilation error on multiple systems if not all